lib/screens/home/home_screen.dart

```diff
@@ -11,53 +29,32 @@ class HomeScreen extends ConsumerStatefulWidget {
 }
 
 class _HomeScreenState extends ConsumerState<HomeScreen> {
+  @override
+  void initState() {
+    super.initState();
+    ref.read(_homeViewModelProvider.notifier).getSurveys();
+  }
+
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      body: HomePagingWidget(surveys: _mockSurveyUiModels),
-    );
+    final surveys = ref.watch(_surveysStreamProvider).value ?? [];
+    final error = ref.watch(_errorStreamProvider).value;
+
+    return ref.watch<HomeState>(_homeViewModelProvider).when(
+          init: () => const HomeLoadingWidget(),
+          success: () => _buildHomeScreen(surveys, false, null),
+          loading: () => _buildHomeScreen(surveys, true, null),
```

Comment:
@ryan-conway just curious, what is the different between init and loading states? 💭

Reply:
`init` should be for the initial loading state with shimmer, and `loading` was planned to be for loading the next page of surveys but this is not yet implemented 🙈 Shall we implement pagination in another task instead, what do you think? CC: @markgravity

Reply:
@ryan-conway yeah, I think we can implement pagination in another task 👍